docs/API/helpers/dynamic-components.md

```
@@ -17,6 +17,160 @@ When the above property is set, the `parent` property of your instance will refe
 
 ## Dynamic container
 
+### `Fliplet.DynamicContainer.get(filter, options)`
+
+```js
+Fliplet.DynamicContainer.get(filter, options)
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)